test/subscribe.js

```diff
@@ -166,7 +198,7 @@ describe('the /subscribe route', function () {
     var SOURCE_URL = 'https://secure.example.com';
     mocks.mockOAuthResponse().reply(200, {
       user: UID,
-      scope: 'basket:write'
+      scope: ['basket']
```

Comment:
These changes are for compatibility with the outh-server /verify API, which returns a list.  The mocked version was incorrectly returning a single string, but  this accidentally still worked because we were using `.indexOf` on it here: https://github.com/mozilla/fxa-basket-proxy/pull/56/files#diff-a70375ad41a72891a73d3ce07c5388efL61

We're now using `.find`, and strings don't have `.find`...